docs/tutorials/java/audiobook/index.md

```diff
@@ -0,0 +1,1005 @@
+
+Once caught up with prerequisites, you can build out your Java project directory.
+
+:::info CAUTIONS
+
+This is a tutorial project and its implementation is suited for personal and hobbyist use.
+In production, you wouldn't read or write from a single database file, hard drive, or system.
+These aren't durable so you can't develop durable software with them.
+You must be able to rebuild your progress state or store information somewhere more durable.
+If you expand on this project, consider a API-based Cloud storage solution.
+
+:::
+
+## Create your Java project structure
+
+1. Set up your project directory:
+   Set up your source code folder hierarchy by issuing this directory creation command:
+
+   ```sh
+   mkdir -p TTSWorker/src/main/java/ttsworker
+   ```
+
+   Your directory structure should now look like this:
+
+   ```text
+   TTSWorker
+   └── src
+       └── main
+           └── java
+               └── ttsworker
+   ```
+
+   You can check it with the Unix `tree` command.
+   If the command isn't native to your system, you can install it with standard package managers like Homebrew and apt.
+
+2. Create your build file
+   In your project root folder, create a build.gradle file and add the following contents.
+   Feel free to swap in Maven if you prefer:
+
+<!--SNIPSTART audiobook-project-java-Gradle-build-file-->
+[build.gradle](https://github.com/temporalio/build-audiobook-java/blob/main/build.gradle)
+```gradle
+group 'ttspackage'; version '0.0.1'
+repositories { mavenCentral() }
+apply plugin: 'java'
+sourceSets.main.java.srcDirs 'src'
+
+dependencies {
+    implementation 'io.temporal:temporal-sdk:1.22.2'
+    implementation 'org.slf4j:slf4j-nop:2.0.6' // logging suppression
+    implementation 'commons-io:commons-io:2.11.0'
+    implementation 'com.squareup.okhttp3:okhttp:4.9.3'
+    implementation 'org.json:json:20210307'
+}
+
+// Run the App
+task run(type: JavaExec) {
+    classpath = sourceSets.main.runtimeClasspath
+    mainClass = 'ttspackage.TTSWorkerApp'
+    standardOutput = System.out
+}
+```
+<!--SNIPEND-->
+
+3. Review your dependencies and run task:
+   Your dependencies include Temporal's [Java SDK](https://github.com/temporalio/sdk-java), and a few basic libraries:
+
+   * **com.squareup.okhttp3:okhttp:4.9.3**: OkHttp is a basic HTTP client for network requests.
+   * **org.json:json:20210307**: Parse and manipulate JSON data.
+   * **commons-io:commons-io:2.11.0**: Perform file tasks with common input/output routines.
+   * **org.slf4j:slf4j-nop:2.0.6**: Minimizes unnecessary output with logging suppression.
+   * **io.temporal:temporal-sdk:1.22.2**: Add error mitigation.
+
+   The run task starts your TTS application.
+
+4. Establish your bearer token environment variable
+   Your application uses the environment variable to authenticate with the OpenAI service.
+   It needs to live in the same shell as  your running app.
+
+   This varies by shell, so you might use the following for tcsh:
+
+   ```sh
```

Comment:
I would swap this with bash. Bash will be more common, so have this be the 'or if you're using a different shell, like tsch'

Reply:
tcsh has been wiped from reality. Resolved.